### PR TITLE
Actor method aliases & isort fix for pre-commit

### PR DESCRIPTION
**Pull request by bandophahita**

### CODE REVIEW

File: tests/test_actor.py [hunk 1]
```diff
@@ -31,9 +31,17 @@ def test_calls_perform_as() -> None:
...
+    actor.will(action)
     actor.should(action)
+    actor.shall(action)
     actor.was_able_to(action)
-    assert action.perform_as.call_count == 3
+    actor.tries_to(action)
+    actor.tried_to(action)
+    actor.tries(action)
+    actor.tried(action)
+    assert action.perform_as.call_count == 11
```
Comment (perrygoy) [inline tests/test_actor.py]: What about something like this?

```python
perform_aliases = [
    actor.did,
    actor.does,
    actor.will,
    actor.should,
    actor.shall,
    actor.was_able_to,
    actor.tries_to,
    actor.tried_to,
    actor.tries,
    actor.tried,
]

for alias in perform_aliases:
    alias(action)

assert action.perform_as.call_count == len(perform_aliases) + 1
```

**Review by perrygoy — DISMISSED** [review]
Haha, i worry a little bit about muddying the `dir()` output, but i can't say i dislike the variety.

I had one small suggestion for the unit tests, but it's OK to merge without it.

File: tests/test_actor.py (hunk 1 above)
Comment (bandophahita) [reply tests/test_actor.py:44]: Good suggestion. I'll update it.

Comment (bandophahita) [reply tests/test_actor.py]: Updated.

**Review by perrygoy — APPROVED** [review]
👍 let the flavor flow!

**Pull request merged**